crates/core/tedge_api/src/entity_store.rs

```diff
@@ -401,7 +444,7 @@ impl EntityStore {
                     external_id: Some(service_external_id),
                     r#type: EntityType::Service,
                     parent: Some(parent_device_id),
-                    other: json!({ "name": service_id,  "type": "systemd" }),
+                    other: json!({ "name": service_id, "type": self.default_service_type }),
```

Comment:
Thing is, if the entity store doesn't add this to the payload, then when the mapper is restarted and `service.type` was changed, then `serviceType` on all the services would change because the mapper would apply the default again.